src/App.js

```diff
@@ -17,13 +17,17 @@ function generateInitialGridState({ width = 1, height = 1}) {
     }
     outerMatrix.push(innerMatrix);
   }
-
   return outerMatrix;
 }
 
+
+
 class App extends Component {
+
   state = {
-    gridLayout: generateInitialGridState({ width: 8, height: 3 })
+    gridLayout: generateInitialGridState({ width: 8, height: 3 }),
+    steps: 8,
```

Comment:
added the amount of steps in the sequence to state...should make this a const somewhere so it can be used by gridLayout and steps.